bin/smoke

```diff
@@ -216,8 +224,7 @@ end
 
 begin
   @options, @args = parse_options
-  application = @args[0]
-  validate_application(application)
+  application = @args[0].split(/ /) # quick hack, does not consider blanks in file names
```

Comment:
ad 1: this is probably most straight forward
ad 2: I liked that the Smoke Smoke tests run on Windows and Linux at the same time without change, that feels portable. When needing to wrap in a batch file this is not possible any more and I would need to duplicate the test suite for Windows. In fact this is necessary right now (before the patch). I just did not think of it. Also for the ASM projects this is not a problem because ASM creates real executables no scripts.
ad 3: could be a good idea anyway. Would it need the command file to be duplicated for each test scenario?